src/main/java/org/apache/wink/rest/ClockinResource.java

```diff
@@ -0,0 +1,146 @@
+import org.apache.wink.common.http.HttpStatus;
+import org.apache.wink.common.model.synd.SyndContent;
+import org.apache.wink.common.model.synd.SyndEntry;
+import org.apache.wink.common.model.synd.SyndFeed;
+import org.apache.wink.common.model.synd.SyndText;
+import org.apache.wink.server.utils.LinkBuilders;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.sql.*;
+
+@Workspace(workspaceTitle = "Demo Bookmarks Service", collectionTitle = "My Bookmarks")
+@Path("bookmarks")
+public class ClockinResource {
+
+    private static final String BOOKMARK          = "bookmark";
+    private static final String SUB_RESOURCE_PATH = "{" + BOOKMARK + "}";
+
+    	
+    @Path("/json")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getJson() {
+    	
+    	Gson gson = new Gson();
+    	
+    	// JSON to Java object, read it from a Json String.
+    	String jsonInString = "{'name' : 'Brent','other' : 'Other String','randomValue' : 12345}";
+    	SimplePOJO json = gson.fromJson(jsonInString, SimplePOJO.class);
+    	
+    	json.setOther("Success");
+
+    	// JSON to JsonElement, convert to String later.
+    	String result = gson.toJson(json);
+    	    	
+    	return Response.status(Response.Status.OK).entity(result).build();	
+    }
+    
+    @Path("/authenticate")
+    @GET
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response isAuth(@QueryParam("username") String username, @QueryParam("password") String password) {
+    	
+    	Status status = Response.Status.OK;
+    	
+    	Gson gson = new Gson();
+    	
+    	// JSON to Java object, read it from a Json String.
+    	if(password == null || password.length() <= 0){
+    		status = Response.Status.FORBIDDEN;
+    	}
+    	String jsonInString = "{'name' : "+username+",'other' : 'Other String','randomValue' : 12345}";
+    	SimplePOJO json = gson.fromJson(jsonInString, SimplePOJO.class);
+    	
+    	json.setOther("Success");
+
+    	// JSON to JsonElement, convert to String later.
+    	String result = gson.toJson(json);
+    	    	
+    	return Response.status(status).entity(result).header("Content-Type", "application/json").build();	
+    }
+    
+    @Path("/database")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getHello() {
+    	
+    	Status status = Response.Status.OK;
+    	Gson gson = new Gson();
+    	ArrayPOJO arr = new ArrayPOJO();
+    	    	
+    	Connection con = null;
+    	try{  
+    		
+    		//step1 load the driver class  
+    		Class.forName("oracle.jdbc.driver.OracleDriver");  
+    		  
+    		//step2 create  the connection object  
+    		con = DriverManager.getConnection(  
+    		"jdbc:oracle:thin:@localhost:1521:xe","system","TonThefloor22T");  
+    		  
+    		//step3 create the statement object  
+    		Statement stmt = con.createStatement();  
+    		  
+    		//step4 execute query  
+    		ResultSet rs=stmt.executeQuery("select * from users");  
```

Comment:
I am using a temporary database with schema:
-The system username and password will need to be changed to your local setup. 
Table: users
Columns: UserId, Lastname, Firstname, Address, City
I have randomly generated three rows worth of data to use to test.